### PR TITLE
UCS/TOOLS: Minor profiling fixes

### PR DESCRIPTION
**Pull request by brminich**
## What
1) Use debug log level for profile enabling message
2) Extend space for func names in `ucx_read_profile`

## Why ?
1) info should be used for important info, need to use it carefully (log is full of profiling messages when profile on scale)
2) Align profile output

Before:
```
    uct_iface_mp_chunk_release        17.142            51          3          uct_mem.c:363  uct_iface_mp_chunk_release()
               ucp_tag_recv_nb         2.049            49         24         tag_recv.c:180  ucp_tag_recv_nb()
ucp_rndv_progress_rma_get_zcopy         2.925            32         11             rndv.c:305  ucp_rndv_progress_rma_get_zcopy()
       ucp_rndv_get_completion         1.959            22         11             rndv.c:440  ucp_rndv_get_completion()
```

After
```
         uct_iface_mp_chunk_release        17.142            51          3          uct_mem.c:363  uct_iface_mp_chunk_release()
                    ucp_tag_recv_nb         2.049            49         24         tag_recv.c:180  ucp_tag_recv_nb()
    ucp_rndv_progress_rma_get_zcopy         2.925            32         11             rndv.c:305  ucp_rndv_progress_rma_get_zcopy()
            ucp_rndv_get_completion         1.959            22         11             rndv.c:440  ucp_rndv_get_completion()
```
Used 35, because of `ucp_eager_offload_sync_ack_handler`, which seems to be the longest func name in UCX being profiled

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10638/ for details (Mellanox internal link).

File: src/tools/profile/read_profile.c [hunk 1]
```diff
@@ -152,28 +152,28 @@ static void show_profile_data_accum(profile_data_t *data, options_t *opts)
     qsort(sorted_locations, num_locations, sizeof(*sorted_locations), compare_locations);
 
     /* Print locations */
-    printf("%30s %13s %13s %10s                FILE     FUNCTION\n",
+    printf("%35s %13s %13s %10s                FILE     FUNCTION\n",
```
Comment (dmitrygladkov) [inline src/tools/profile/read_profile.c]: what do you think to define macro for this value:
```
#define UCS_PROFILE_MAX_NAME_LEN   35
```

and update `UCS_PROFILE_*` macros to check that the name is `<=` than `UCS_PROFILE_NAME_FORMAT`, e.g.
```
#define UCS_PROFILE_FUNC(_ret_type, _name, _arglist, ...) \
    static UCS_F_ALWAYS_INLINE _ret_type _name##_inner(__VA_ARGS__); \
    \
    _ret_type _name(__VA_ARGS__) { \
        ucs_assert(strlen(_name) <= UCS_PROFILE_MAX_NAME_LEN); \
        UCS_PROFILE_SCOPE_BEGIN(); \
        _ret_type _ret = _name##_inner _arglist; \
        UCS_PROFILE_SCOPE_END(#_name); \
        return _ret; \
    } \
    static UCS_F_ALWAYS_INLINE _ret_type _name##_inner(__VA_ARGS__)

```

So, if the newly added function name is greater than this value, a developer could change `UCS_PROFILE_MAX_NAME_LEN` to the new value to fit new name

Comment (brminich) [reply src/tools/profile/read_profile.c]: sounds, good. But maybe we can just cut the name by MAX_NAME_LEN to keep the table format accurate?
I guess 35 symbols would be enough to recognize the func.

Comment (dmitrygladkov) [reply src/tools/profile/read_profile.c]: > But maybe we can just cut the name by MAX_NAME_LEN to keep the table format accurate?

yes, good idea :)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10644/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10647/ for details (Mellanox internal link).

**Pull request merged**